docs/removeFlowTypesLoader.js

```diff
@@ -2,24 +2,37 @@
 const flowRemoveTypes = require('flow-remove-types');
 const prettier = require('prettier');
 
+/**
+ * This Webpack loader is used to clean up the docs code examples for display in the Sandpack code editor. It removes Flow types and eslint disable comments.
+ */
 module.exports = function removeFlowTypesLoader(source /*: string */) {
   const callback = this.async();
   this.cacheable();
 
   const options = {};
-  const removed = flowRemoveTypes(source).toString().replace('//       strict', '');
+  // Remove Flow declarations/types
+  const removed = flowRemoveTypes(source)
+    .toString()
+    // Also remove what's left of the initial declaration comment
+    .replace('//       strict', '')
+    // Remove eslint disable comments
```

Comment:
This doesn't cover block-level disables. We don't currently have the need, but this can be updated if we do in the future